src/vm/checker/typecheck/tests/mod.rs

```diff
@@ -13,6 +13,24 @@ fn type_check(exp: &SymbolicExpression) -> TypeResult {
     type_checker.type_check(exp, &contract_context)
 }
 
+#[test]
+fn test_get_block_info(){
```

Comment:
I think we need a better name for this built-in, so developers don't confuse it with querying Stacks block information.  Maybe we can call it `get-burn-block-info`?

Reply:
@jcnelson and I discussed and decided to leave this call as implicitly appearing query a Stacks block -- where any info that is currently only in the burn chain block (the block time for now) is transparently "hoisted".

In the future, as dependency on the burn chain decreases, various block properties like this may actually end up coming from Stacks blocks.